### PR TITLE
OpenAI dependencies tier 2

### PR DESCRIPTION
**Pull request by Schala**
These packages have dependencies on those committed in tier 1. I'll do my best to keep hese updated. They are as follows:

- python-ffmpeg-python (pypi package is ffmpeg-python)
- python-fire
- python-flask
- python-labelme
- python-numba
- python-oauth2client
- python-pytest-localserver
- python-responses
- python-scikit-image
- python-yarl

### CODE REVIEW

**Comment by lazka** [comment]
tier 1 is in the repo now. I've retriggered CI here and there are some errors.

**Comment by Schala** [comment]
Okay yeah I see it. Looks like pytest-localserver 404s. I'll take a look.

**Comment by lazka** [comment]
One thing I forgot last time, this:

```
provides=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
conflicts=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
replaces=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
```

is not needed anymore. It was there to let old packages transition from python3 to python naming.

Can you remove them in all packages?

**Comment by Schala** [comment]
Sure, I'll do the same for tier 1 as well. There are other packages that aren't mine that have this as well.

**Comment by lazka** [comment]
> Sure, I'll do the same for tier 1 as well. There are other packages that aren't mine that have this as well.

No need to touch the old ones imo.

File: mingw-w64-butler/PKGBUILD [hunk 1]
```diff
@@ -1,27 +0,0 @@
-# Maintainer: Sarah Ottinger <schalaalexiazeal@gmail.com>
```
Comment (lazka) [inline mingw-w64-butler/PKGBUILD:1]: Why did you delete this?

Comment (Schala) [reply mingw-w64-butler/PKGBUILD:1]: Butler wasn't part of the OpenAI branch. I had it committed to master.

Comment (lazka) [reply mingw-w64-butler/PKGBUILD]: ah, sry, I missed that. I've rebased this now on top of current master.
